From 0ce15623052cb0f5704044e4967781cdb5876ae5 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 20 Oct 2022 02:37:21 +0000
Subject: [PATCH] aptiv-s32g: linux: disable flexcan3

The flexcan3 will be used by M7, so disable it in dts.

Upstream-Status: Pending

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32g274a-aptiv.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/s32g274a-aptiv.dts b/arch/arm64/boot/dts/freescale/s32g274a-aptiv.dts
index d711a07e2569..e2923ee10d70 100755
--- a/arch/arm64/boot/dts/freescale/s32g274a-aptiv.dts
+++ b/arch/arm64/boot/dts/freescale/s32g274a-aptiv.dts
@@ -128,7 +128,8 @@ &can2 {
 &can3 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&can3_pins>;
-	status = "okay";
+	/* flexcan3 is used by M7 */
+	status = "disabled";
 };
 
 &stm0 {
-- 
2.32.0

